.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+sudo: required
```

Comment:
Forgot to mention the link: http://docs.travis-ci.com/user/trusty-ci-environment/